joomla/joomla-cms
# Convert mod_stats to service provider

### Summary of Changes

Converts the statistics module to service provider.


### Testing Instructions

1. Make sure you have site module mod_stats published
2. You should see module output on frontend

![mod_stats](https://github.com/joomla/joomla-cms/assets/59166858/33eb0e83-f696-4278-9b17-5ed05a9db1e8)

### Actual result BEFORE applying this Pull Request

Module works as expected

### Expected result AFTER applying this Pull Request

Module works same way as before applying pull request

### Link to documentations
Please select:
